tests/framework/CodeInterfaceTests/Neutrino/gold/Neutrino_Base/sampleNeutrino/1/NeutrinoInput.nescene

```diff
@@ -0,0 +1,945 @@
+<!--For the program to work correctly, creations should come first, then the connections, and finally the properties--><nescene version="0.4">
```

Comment:
you should remove this... you are not testing against this... you just test _"Neutrino_Base/out_dump.csv"_ in your **tests** file